### DISCUSSION TITLE
승객 생성량과 관련된 이벤트

### DISCUSSION THREAD

**Discussion by mille-feu** [Tiny-Metro/Poly-Metro#20] [Ideas]
여러 이벤트를 통해, 승객의 생성량이 변화할 수 있습니다.
- 해당 이벤트들을 통해 증, 감소하는 수치가 합적용인지, 곱적용인지
- 계산된 최종 수치가 소수점일 경우, 어떻게 처리할 지
  - ex) 현재 생성량이 6 People/sec일 때, 10% 증가할 경우 6.6 People/sec 가 된 경우의 처리

관계된 다른 논의사안 : [승객 생성 알고리즘](https://github.com/Tiny-Metro/Tiny-Metro/discussions/18)

**Comment by cstrnull00** [comment]
* 곱 적용
* 확률
  * 역의 승객은 기본적으로 6초당 66.666%의 확률로 1명이 생성됨. (1/9 명 / sec)
  * 이와 같이 10% 증가했을 경우 6초당 73.333%의 확률로 1명이 생성됨 (11/90명 / sec)
  * 이에 더해 20% 증가한 이벤트가 발생했을 경우 88%의 확률로 6초당 1명이 생성됨 (132/900명 / sec)
